tusk_11py.3.py

```diff
@@ -0,0 +1,14 @@
+class CustomContextManager:
+    def __enter__(self):
+        print("=" * 10)
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        print("=" * 10)
+        return self
+
+    def exit(self, ex_type, ex_value, traceback):
+        print("=" * 10)
+        if exit() is not None:
```

Comment:
`exit()` is a build-in function that stops a program. I doubt you want to stop a program at this point